### DISCUSSION TITLE
help please

### DISCUSSION THREAD

**Discussion by pheonex171** [aclap-dev/video-downloadhelper#1451] [Help and Q&A]
### Your question

i downloaded videos using the co app and it says that they downloaded and has te play ico.
i went looiing and i cant find them, the defaukt folder is still listed as dwhelper but the folder is empty with 0 files in it.

linux: ubuntu 64 os
browser: firefox

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"/home/serenity/.local/share/vdhcoapp/vdhcoapp","version":"2.0.19","new_version":false}
license: {"unneeded":true}
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Linux x86_64; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 15,
    "successfull_dl": 19
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Download history is kept by VDH but you have to enable it.  By default the history is NOT kept.  Open the VDH menu & hover your mouse over all the icons.  Read the tooltips as they hover up.  You'll find the history feature soon enough.  It looks like you have not enabled history so this wonderful feature is not going to help you.  I advise you to turn it on right now.  Unfortunately, it is too late to help you with your current problem.

Here is something that might -- MIGHT -- tell you where your downloads went.  Pick any video to download.  But DO NOT do a simple `Download` function in VDH.  Do a `Download As` function instead.  Let it open the file selection dialog.  Make a note of the directory the file selection dialog proposes.  Cancel the download.  Go find that directory.  With any luck, that's where your downloads went.

To find the `Download As` function, look at the blue control for downloading in the VDH menu.  You will notice that it's actually 2 controls side by side.  There's the word `Download` & then to its right there's a small D-shaped button with 3 vertical dots in it.  Click that D-shaped button.  That opens a secondary menu.  You will find several options on that secondary menu, `Download As` among them.  But make yourself familiar with all the choices on that secondary menu.  You never know when one of them might come in handy.  My preference is to make `Download As` my default action in VDH.  There is an option on that secondary menu for changing your default action.  Look for it.  Experiment with it.  You'll learn how it works soon enough.

If starting but interrupting a `Download As` doesn't tell you where your downloads went, you will be forced to do some sort of file search on your system.  My preference is to use the dir command from a command line but there are other techniques for searching for files.  Use whatever you are comfortable with.

**Reply by GrampaWildWilly** [reply]
Forgot to mention the history retention period.  There is a setting in the VDH `More settings` dialog that sets the number of days VDH will retain history entries before purging the oldest ones.  Scroll the settings down & you'll find it soon enough.
